core/renderer/shaders/positionTextureGrayAlpha.frag

```diff
@@ -0,0 +1,16 @@
+#version 310 es
+precision highp float;
+precision highp int;
```

Comment:
Use lowp precision as default for both float and int. That's enough for simple color processing.

Reply:
This shader was copied from `positionTextureColor.frag`, with the only section changed being what is in the `main()` function.  The existing shaders use the same settings for precision etc..

Reply:
I'm aware that all shaders for some reason were marked with highp, and I've actually benchmarked that it has a noticeable performance impact on mobile devices.

Reply:
I didn't realise there was such a performance hit on mobile devices, so if there is no need to have them marked as `highp`, then at some point we should adjust the rest of the shaders as well.